README.md

```diff
@@ -34,7 +34,8 @@ The gem also provides server-side support for Turbolinks redirection.
 Your application can use the [`turbolinks` npm package](https://www.npmjs.com/package/turbolinks) to install Turbolinks in a [Webpack](http://webpack.github.io/) asset bundle.
 
 1. Add the `turbolinks` package to your application: `npm install --save turbolinks`.
-2. Add `turbolinks` to the `entry` section of webpack.config.js:
+2. Install the [imports loader](https://github.com/webpack/imports-loader) module for webpack: `npm install --save imports-loader`. 
```

Comment:
either `--save` or `--save-dev` depending on if you need this for your production environment. I'd say it's fine as-is.